masterlist.yaml

```
@@ -6006,3 +6006,10 @@ plugins:
     clean:
       - crc: 0x489330C5
         util: 'FO4Edit v4.0.3h (Hotfix 1)'
+
+  - name: 'Omniguous-no-useless-papers.esp'
+    url: [ 'https://www.nexusmods.com/fallout4/mods/52722' ]
+    req: [ 'Unofficial Fallout 4 Patch.esp' ]
```

This can be removed as it's already a master file for this module.
```suggestion
    req: [ 'Unofficial Fallout 4 Patch.esp' ]
```